README.md

```diff
@@ -10,14 +10,15 @@ The Node.js End User Feedback team works with Node.js end users to provide a bi-
 * Support [CommComm](https://github.com/nodejs/community-committee) and [TSC](https://github.com/nodejs/community-committee) end user feedback and outreach needs.
 
 ### Governance and Contributing
-The Node.js End User Feedback team has adopted the core governance and contributing policies of Node.js.
+The Node.js End User Feedback team has adopted the core governance and contribution policies of the Node.js project.
```

Comment:
>  User Feedback team

This is an initiative, not a team - I believe team is a (possibly outdated) term used by the TSC and may be confusing if used here. I suggest changing this to `User Feedback Initiative`. 👍